lib/ns.js

```diff
@@ -15,13 +15,15 @@ export function parseName(name, defaultPrefix) {
   if (parts.length === 1) {
     localName = name;
     prefix = defaultPrefix;
-  } else
+  }
 
   // prefix + local name
-  if (parts.length === 2) {
+  else if (parts.length === 2) {
     localName = parts[1];
     prefix = parts[0];
-  } else {
+  }
+
+  else {
```

Comment:
TBH I still find this construct confusing. This is how I'd write this:

```suggestion
  } else if (parts.length === 2) { 
  
    // prefix + local name
    localName = parts[1];
    prefix = parts[0];
  } else {
```

Reply:
You'd consistently go for 

```
if (a) {
  ...
} else if (b) {
  ...
} else {
  ...
}
```

I guess we'd need to move comments _into_ the blocks then?

```
if (a) {

  // very complicated (!)
  ...
} else if (b) {

  // also complicated (!)
  ...
} else {

  // simple enought
  ...
}
```